### ISSUE TITLE
Performance comparison of understand_sentiment model in Fluid and Paddle V2 (GPU version)

### ISSUE THREAD

**Issue by ranqiu92** [PaddlePaddle/Paddle#7046]
Training Set Pass accuracy

| Pass | Paddle V2 | Fluid|
|------|-----------|------|
|0	| 0.83468 | 0.8312 |
|1	| 0.90696	| 0.89579 |
|2	| 0.9388 | 0.92891 |
|3	| 0.96404 | 0.95559 |
|4	| 0.983 | 0.9788 |
|5 | 0.99064 | 0.99043 |
|6	| 0.99544	| 0.99407 |
|7 | 0.99716	| 0.99519 |
|8 | 0.99692 | 0.99711 |
|9	| 0.99628	| 0.9982 |

Test set accuracy

| Pass | Paddle V2 | Fluid |
|----|-----------|-------|
|0	| 0.87836 | 0.8696 |
|1	| 0.89091 | 0.88339 |
|2	| 0.88772 | 0.88537 |
|3	| 0.88567 | 0.87996 |
|4	| 0.87432 | 0.87703 |
|5	| 0.88031 | 0.87267 |
|6	| 0.88111	| 0.87703 |
|7	| 0.8782 | 0.87992 |
|8	| 0.87764 | 0.87967 |
|9 | 0.87968 | 0.8768 |

reference #6650
